### ISSUE TITLE
Wonder about proxyNCA loss

### ISSUE THREAD

**Issue by YanhuaCheng** [dichotomies/proxy-nca#15]
loss = torch.sum(- T * F.log_softmax(D, -1), -1), should D be -D since we want to minimize the distance between the example with its proxy with same label.

**Comment by dichotomies** [comment]
Thanks for pointing that out. I have omitted the negative sign by accident. Interestingly, the results don't differ with positive distances though.

I have fixed this issue in the most recent commit.
